src/test/regress/sql/multi_alter_table_add_constraints_without_name.sql

```diff
@@ -0,0 +1,227 @@
+--
+-- MULTI_ALTER_TABLE_ADD_CONSTRAINTS_WITHOUT_NAME
+--
+-- Test checks whether constraints of distributed tables can be adjusted using
+-- the ALTER TABLE ... ADD without specifying a name.
+
+SET citus.shard_count TO 32;
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 5410000;
+ALTER SEQUENCE pg_catalog.pg_dist_placement_placementid_seq RESTART 5410000;
+
+CREATE SCHEMA sc1;
```

Comment:
We usually pick a more descriptive schema name, same goes for arbitrary configs test.

Reply:
Changed it to AT_AddConstNoName